### PR TITLE
Updating using-tcp-services.md to remove deprecated message

### LINKED ISSUE #11836: TcpListener constructor is deprecated
This article is creating an instance of `TcpListener` with a deprecated constructor. It might be a good idea to update this article to use the current version. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cb4eec82-3aed-b326-1927-9f3d606f9d6b
* Version Independent ID: 4cabdedf-e7e6-4964-2e7e-521d2ca63f48
* Content: [Using TCP Services](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/using-tcp-services)
* Content Source: [docs/framework/network-programming/using-tcp-services.md](https://github.com/dotnet/docs/blob/master/docs/framework/network-programming/using-tcp-services.md)
* Product: **dotnet-framework**
* Technology: **dotnet-networking**
* GitHub Login: @karelz
* Microsoft Alias: **ncldev**

### PR DESCRIPTION
**Pull request by RickAcb**
Solved the issue of TcpListener constructor is deprecated see issue #11836

Fixes #11836

### CODE REVIEW

File: docs/framework/network-programming/using-tcp-services.md [hunk 1]
```diff
@@ -159,7 +161,7 @@ public class TcpTimeServer {
     public static int Main(String[] args) {  
         bool done = false;  
   
-        TcpListener listener = new TcpListener(portNum);  
+        TcpListener listener = new TcpListener(IPAddress.Any, portNum);
```
Comment (Youssef1313) [inline docs/framework/network-programming/using-tcp-services.md]:
```suggestion
        var listener = new TcpListener(IPAddress.Any, portNum);  
```

File: docs/framework/network-programming/using-tcp-services.md [hunk 2]
```diff
@@ -159,7 +161,7 @@ public class TcpTimeServer {
     public static int Main(String[] args) {
```
Comment (Youssef1313) [inline docs/framework/network-programming/using-tcp-services.md]:
```suggestion
    public static int Main(String[] args)
    {
```

File: docs/framework/network-programming/using-tcp-services.md [hunk 3]
```diff
@@ -100,7 +100,8 @@ public class TcpTimeClient {
  The following example demonstrates creating a network time server using a **TcpListener** to monitor TCP port 13. When an incoming connection request is accepted, the time server responds with the current date and time from the host server.  
   
 ```vb  
-Imports System  
+Imports System
```
Comment (Youssef1313) [inline docs/framework/network-programming/using-tcp-services.md]:
```suggestion
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

**Review by BillWagner — APPROVED** [review]
This LGTM. I've resolved the conflicts, and I'll :shipit: when the build is clean.

**Pull request merged**